docs/supervisor/development.md

```diff
@@ -21,6 +21,73 @@ The instructions here is for development of the Supervisor itself.
 When the initializing is complete you can access the Home Assistant frontend on `http://localhost:9123`
 
+
+If you need to rebuild the base of the Supervisor run the task "Build Supervisor" and then restart the "Run Supervisor" task again when that completes.
+
+#### Using Windows WSL2
+
+While the system is Linux and compatible with Mac, it is also possible to build and test locally on Windows using WSL2 with Debian and other similar distros. This is achieved via Windows Subsystem for Linux 2 (WSL2), which makes POSIX tasks possible within Windows via system-level integration with a Linux virtual machine.
+
+:::tip
+You must use WSL2. WSL version 1 is not capable of running Docker and therefore is incompatible. If using this method, you must ensure you have [set up WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+:::
+
+It is assumed at this point you have installed Debian from the Microsoft Store and [Visual Studio Code](https://code.visualstudio.com/download). From within your WSL2-enabled Debian terminal, run the following annotated bash script. If you copy the entire script it should perform the required setup of Debian for Home Assistant development but any problems may be difficult to troubleshoot, so it is recommended to run this line-by-line.
+
+```bash
+apt update; apt install -y curl wget git gnupg; #Update and install our dependencies
+curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash; #Install NVM
+source ~/.profile; nvm install node; #reload profile and install node
+sudo apt remove cmdtest yarn  #Remove problematic packages if installed. Debian current Yarn does not work.
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -  #Add yarn repo certificate
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list #Add Yarn repo
+sudo apt update; sudo apt install yarn -y; #Install yarn
+curl -fsSL https://get.docker.com |sudo bash  #install docker Community Edition
+sudo update-alternatives --set iptables /usr/sbin/iptables-legacy #Fix IP Tables for Docker
+git clone https://github.com/home-assistant/supervisor /home/$USER/supervisor #Clone Supervisor to ~/supervisor
+usermod -aG docker $USER #Add self to docker group
+systemctl enable docker.service #enable docker start on boot
+service docker start #Start the docker service
+service docker status #Verify docker status is running
+```
+
+Restart your Debian terminal to activate the new profiles and software configurations. 
+
+:::tip
+Always run scripts line by line to figure out where the problem lies. If you need to completely start over, you can uninstall the Debian app, then from the Windows command prompt execute `del %HOMEPATH%\AppData\Roaming\Code`. After removing your VSCode settings, launch VS Code, and remove all extensions. Finally either execute `net stop LxssManager` from an administrative `cmd` prompt or reboot your computer and try again.
+:::
```

Comment:
This looks like a really brute force kill all your setting approach. This will break other things in your system. I really don't want to promote such actions.

Reply:
What do you recommend?

Reply:
breaking peoples vscode setups is not acceptable.